src/kafka-util/src/client.rs

```diff
@@ -438,7 +444,30 @@ impl<C> ClientContext for TunnelingClientContext<C>
 where
     C: ClientContext,
 {
-    const ENABLE_REFRESH_OAUTH_TOKEN: bool = C::ENABLE_REFRESH_OAUTH_TOKEN;
+    const ENABLE_REFRESH_OAUTH_TOKEN: bool = true;
+
+    fn generate_oauth_token(
```

Comment:
Is it worth a comment here indicating that the only oath we support will be for AWS?

Reply:
Sure! Done.